static/js/avatar.js

```diff
@@ -58,16 +61,29 @@ export function build_user_avatar_widget(upload_function) {
     $("#user-avatar-upload-widget .image-delete-button").on("click keydown", (e) => {
         e.preventDefault();
         e.stopPropagation();
-        channel.del({
-            url: "/json/users/me/avatar",
-            success() {
-                $("#user-avatar-upload-widget .image-delete-button").hide();
-                $("#user-avatar-source").show();
-                // Need to clear input because of a small edge case
-                // where you try to upload the same image you just deleted.
-                get_file_input().val("");
-                // Rest of the work is done via the user_events -> avatar_url event we will get
-            },
+        function delete_user_avatar() {
+            channel.del({
+                url: "/json/users/me/avatar",
+                success() {
+                    $("#user-avatar-upload-widget .image-delete-button").hide();
+                    $("#user-avatar-source").show();
+                    // Need to clear input because of a small edge case
+                    // where you try to upload the same image you just deleted.
+                    get_file_input().val("");
+                    // Rest of the work is done via the user_events -> avatar_url event we will get
+                },
+            });
+        }
+        const modal_parent = $("#account-settings");
+
+        const html_body = render_confirm_delete_user_avatar();
+
+        confirm_dialog.launch({
+            parent: modal_parent,
+            html_heading: i18n.t("Delete profile picture"),
+            html_body,
```

Comment:
If I would not have made template I would have to write the html in this file which I do not find consistent with other places where we use confirm_dialog.

Reply:
Can't we pass the text as a simple `i18n` string, just like you do for `html_heading`?

Reply:
I think we can do this. But I see similar templates like <code>confirm_delete_user.hbs</code>. Maybe we used templates there as we were sending user data along with body. But I am not sure that having strings without any tags in body is the right way. What do you think?

Reply:
Yeah, I think template is used as `confirm_delete_user.hbs` because the name needs to be passed (and I guess shown as bold, which is difficult to do without using any HTML). Here, I feel using a template would be an avoidable complexity, since its a fixed string.

Reply:
Yeah, we generally prefer to use templates for HTML and avoid generating it in JS wherever we can -- we do have the `<p>` tag that makes this not just a simple string.

Reply:
Got it, thanks for the clarification!